scripts/test/test_util.js

```diff
@@ -177,13 +177,14 @@ var CommandlineTest = function(command) {
             flushAndFail('expecting test max duration: ' +
                 self.expectedMaxTestDuration_ + ', actual: ' + duration);
       }
-    }).fin(function() {
```

Comment:
`fin` is equivalent to `Promise.finally`, can we use that instead of `catch`ing?